tests/dummy/app/templates/getting-started/assets.hbs

```diff
@@ -1,4 +1,16 @@
 {{! template-lint-disable block-indentation }}
+<h1 id="included-assets">Included Assets</h1>
+
+<p>ember-bootstrap uses the Bootstrap CSS classes while replacing the behaviors from the components Bootstrap implements
+    in <code>bootstrap.js</code>, such as toggle, navbar, modal, etc., with equivalent, CSS class-compatible
+    native Ember components.</p>
+
+<p>If you are not using a preprocessor, ember-bootstrap will import the static CSS files. If you are using a preprocessor
+    like LESS or SASS, ember-bootstrap imports the preprocessor-ready versions of the styles.</p>
+
+<p>ember-bootstrap specifically excludes `bootstrap.js`. The jQuery and Ember ways of control flow and animation sometimes
```

Comment:
Hm, I merged this probably a bit too early... @srvance Feel free to reopen if you think those changes make sense.